Units/docker-images/once.sh.ubuntu.22.04-builder/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use an official Python runtime as a parent image
-FROM ubuntu:22.04
+FROM ubuntu:latest
```

Comment:
Is the fix from `once.sh.ubuntu.22.04-server` needed here as well? --> "# FROM ubuntu:latest # doesn't work on arm64 platform with DOCKER_BUILDKIT"

Reply:
This is the builder Dockerfile with the latest ubuntu image. This image is not part of a multi platform build, so no problem.